docs/server-side/v2/changelog/release-notes.mdx

```diff
@@ -5,6 +5,9 @@ nav: 5.1
 
 This Changelog highlights notable changes to the 100ms server-side API, such as API additions, improvements, and deprecations. Also, we've included developer experience improvements to this page to keep you on track with items that will enhance your integration journey.
 
+## 2023-10-31
+- Added support for setting `max_duration_seconds` in [Room API](/server-side/v2/api-reference/Rooms/overview).
```

Comment:
Can we also add what it does?
"This will allow you to set the maximum duration of a running session before it times out and the session is closed."